canonical-websites/jaas.ai
# Add Prettier ✨

## Done

- Added Prettier as `pre-commit` hook
- Ran Prettier over `.js` files
- Ran Prettier over `.scss` files

## QA

- Run `./run`
- Check everything looks hunky-dory
